.env.template.mainnet.eon

```
@@ -3,8 +3,25 @@ COMPOSE_PROJECT_NAME=evmapp-mainnet-eon
 INTERNAL_NETWORK_SUBNET='10.10.50.0/24'
 SCNODE_IP_ADDRESS='10.10.50.3'
 
+ZEN_TAG=v4.1.1
+ZEN_PORT=19033
```

```suggestion
ZEN_PORT=9033
```
Let's use network specific default ports.